features/admins/admin_post_news.feature

```diff
@@ -8,6 +8,10 @@ Feature: Admin Actions to Post News
     Given I am logged in as a "tag_wrangling" admin
     When I go to the admin-posts page
     Then I should not see "Post AO3 News"
+    Given I am logged in as a "communications" admin
+    When I go to the admin-posts page
+    Then I should see "AO3 News"
+      And I should see "Post AO3 News"
```

Comment:
I think we can update "Scenario: Log in as an admin and create an admin post with tags" instead, like so:
```
    Given I am logged in as a "communications" admin
    When I follow "Admin Posts"
    Then I should see "AO3 News" within "#main ul.navigation"
      And I should see "Post AO3 News" within "#main ul.navigation"
    When I follow "Post AO3 News" within "#main ul.navigation"
    Then I should see "New AO3 News Post"
```

(As with the Known Issues test, I didn't check that the selectors I used here are correct; they might need some work. But using them is a good idea because the text we're looking for appears in multiple locations.)